netlify.toml

```diff
@@ -1,7 +1,7 @@
 [build]
   base      = "site/"
   publish   = "site/dist"
-  command   = "gulp build"
+  command   = "npm run build"
```

Comment:
I'd say `npm run` adds the local node_modules/.bin folder to PATH but netlify does it too

From https://www.netlify.com/docs/build-settings/:
> All of the installing of dependencies described here happens before your build command is run — so you can include webpack in your /package.json and be confident that it will be npm install‘ed before we run your build command webpack!

I think we have to use `npm scripts` for this kind of commands. If anyone try to reproduce `gulp build` in their term, it wont work out of the box, they will have to do either
- `npx gulp build`
- `./node_modules/.bin/gulp build`

but `npm run build` will work

Reply:
Step 3 in the development getting started guide is to install gulp globally 🤔 The reason I don't like using proxy methods in cases like this is it adds unnecessary bloat and, depending on your system, can hurt performance

Reply:
I reworked the doc in here: https://github.com/swapagarwal/swag-for-dev/pull/76/files#diff-d432571d5bbb8bd98a141819a9805ef5R77, still waiting for being rebased and merged.

Do you have such examples of performance issues using `npm run` proxy?

As netlify adds a path for this command to work I assume we can let it like this but I think we need to update the doc to use `npm run build`. We could still add a note about gulp cli.

Reply:
That's specifically for Docker, right? Since there are issues with integrating gulp + docker together?

I don't have examples, but from experience with WSL, the npm command would have to run, which includes running the npm-cli codebase, and then the gulp codebase needs to run (as it would be called from npm)

I just ran `time npm -v` in WSL and it took  ~4s, with `time gulp -v` taking ~8s

Running it again after files have been cached, the performance is better, with ~1.5s for both commands combined

Reply:
It's in the Docker PR but in the `Production/Local` section of the file. 
https://github.com/swapagarwal/swag-for-dev/blob/2815533d35d1bbe3681825b5ef7c9228e0f6bd73/site/README.md#local-1

Didn't know about this these lags, gonna commit this last change so we can merge this PR.

Reply:
This can definitely be a discussion we have in gitter 😛